contexts/xmtp.ts

```diff
@@ -9,26 +8,15 @@ export type MessageStoreEvent = {
 }
 
 export type XmtpContextType = {
-  wallet: Signer | undefined
-  walletAddress: string | undefined
   client: Client | undefined
   conversations: Conversation[]
   loadingConversations: boolean
-  getMessages: (peerAddress: string) => Message[]
-  dispatchMessages?: Dispatch<MessageStoreEvent>
-  connect: (wallet: Signer) => void
-  disconnect: () => void
```

Comment:
I like this change of breaking out the contexts so that we can require them more granularly